src/Field/Email.php

```diff
@@ -84,14 +88,31 @@ public function normalize($value)
         return parent::normalize(implode(', ', $emails));
     }
 
+    /**
+     * Validate DNS.
+     */
+    protected function isDNSValid(string $domain): bool
+    {
+        return $this->hasDNSRecord($domain, true) || $this->hasDNSRecord($domain, false);
+    }
+
+    private function hasDNSRecord(string $domain, bool $isMX): bool
+    {
+        $normalizedDomain = $domain . '.';
+        if (!checkdnsrr($normalizedDomain, ($isMX ? 'MX' : 'A'))) {
```

Comment:
Do we need `checkdnsrr` at all?

Reply:
Not sure, but I guess it's good to check and step out of method early.
Probably get_dns_records() takes more time and resources to execute.